vault/provider.go

```diff
@@ -103,6 +103,7 @@ func Provider() terraform.ResourceProvider {
 			"vault_aws_auth_backend_sts_role":           awsAuthBackendSTSRoleResource(),
 			"vault_aws_secret_backend":                  awsSecretBackendResource(),
 			"vault_aws_secret_backend_role":             awsSecretBackendRoleResource(),
+			"vault_consul_role":                         consulRoleResource(),
```

Comment:
Could this be renamed to `vault_consul_secret_backend_role`? I just merged a PR (https://github.com/terraform-providers/terraform-provider-vault/pull/59) adding a Consul secret resource that manages the config and it would be great if it looked like these two came in together.